Dockerfile

```
@@ -113,7 +113,8 @@ EOF
 # runtime stage: produces final image that will be executed
 FROM base-target-stage
 
-LABEL io.cartesi.sdk_version=0.9.0
+LABEL io.cartesi.sdk_name=cryptobughunters/sdk
```

maybe we could specify `docker.io` or `ghcr.io`

For ex.:

```suggestion
LABEL io.cartesi.sdk_name=ghcr.io/cryptobughunters/sdk
```

---

yes, by default it's docker.io

it's not mandatory, I'm just favoring to be more specific, it's up to you